packages/integration_test/pubspec.yaml

```diff
@@ -1,10 +1,10 @@
 name: integration_test
 description: Runs tests that use the flutter_test API as integration tests.
-version: 0.9.2+1
+version: 0.10.0-nullsafety
 homepage: https://github.com/flutter/plugins/tree/master/packages/integration_test
 
 environment:
-  sdk: ">=2.2.2 <3.0.0"
+  sdk: '>=2.10.0-56.0.dev <3.0.0'
```

Comment:
I wonder if this will break the flutter/engine tests, when we want to update e2e version?

https://github.com/flutter/engine/blob/master/e2etests/web/regular_integration_tests/pubspec.yaml#L5 For integration tests

web_ui: https://github.com/flutter/engine/blob/master/lib/web_ui/pubspec.yaml#L5

Reply:
There's no rush on this, and we won't be able to move forward with it realistically until package:vm_service is migrated, which probably won't happen for at least a couple more weeks.

This constraint _should_ be ok, but by the time we'd publish this it should have a real 2.11 (not .dev) constraint.

Reply:
thanks for info! :)